modules/simple-payments/simple-payments.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+class Jetpack_Simple_Payments {
+	// These have to be under 20 chars because that is CPT limit.
+	static $post_type_order = 'jp_pay_order';
+	static $post_type_product = 'jp_pay_product';
+
+	static $shortcode = 'simple-payment';
```

Comment:
Yes, but the shortcode represents a single simple payment, so it would be weird to have:
```
[simple-payments id=43]
```